### PR TITLE
GET/SET user Capacity usage F-71A.

### PR DESCRIPTION
**Pull request by amit-si-seagate**
# Problem Statement
- GET/SET IAM user API for Capacity usage F-71A.

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by aditipande96** [comment]
@amit-si-seagate Pls resolve the codacy issues.

File: tests/csm/rest/test_capacity_quota.py [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+        self.bucket = "iam-user-bucket-" + str(int(time.time_ns()))
+        self.obj_name_prefix = "created_obj"
+        self.obj_name = "{0}{1}".format(self.obj_name_prefix, perf_counter_ns())
+        self.display_name = "iam-display-name-" + str(int(time.time_ns()))
+        #self.log.info("Verify Create bucket: %s with access key: %s and secret key: %s",
+        #              self.bucket, self.akey, self.skey)
+        self.cryptogen = SystemRandom()
+        #assert s3_misc.create_bucket(self.bucket, self.akey, self.skey), "Failed to create bucket."
+
+    def get_iam_user_payload(self, param=None):
```
Comment (aditipande96) [inline tests/csm/rest/test_capacity_quota.py:90]: Common functions can be moved to lib file libs/csm/rest/csm_rest_quota.py

File: tests/csm/rest/test_capacity_quota.py [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+        resp = self.csm_obj.create_iam_user_rgw(payload)
+        self.log.info("Verify Response : %s", resp)
+        assert_utils.assert_true(resp.status_code == HTTPStatus.CREATED,
+                                 "IAM user creation failed")
+        self.user_id = resp.json()['tenant'] + "$" + payload["uid"]
+        self.created_iam_users.add(self.user_id)
+        resp1 = self.csm_obj.compare_iam_payload_response(resp, payload)
+        self.log.info("Printing response %s", resp1)
+        assert_utils.assert_true(resp1[0], resp1[1])
+        #self.akey = resp.json()["access_key"]
```
Comment (aditipande96) [inline tests/csm/rest/test_capacity_quota.py:79]: Pls remove this commented part if not needed.

File: libs/csm/csm_interface.py [hunk 3]
```diff
@@ -29,10 +29,10 @@
 from libs.csm.rest.csm_rest_s3user import RestS3user
 from libs.csm.rest.csm_rest_stats import SystemStats
 from libs.csm.rest.csm_rest_system_health import SystemHealth
-
+from libs.csm.rest.csm_rest_quota import GetSetQuota
```
Comment (kachhwahadivya) [inline libs/csm/csm_interface.py:32]: This are changes from main. can u please rebase

Comment (amit-si-seagate) [reply libs/csm/csm_interface.py:32]: Sure @kachhwahadivya !

**Comment by amit-si-seagate** [comment]
Re-opened New PR: https://github.com/amit-si-seagate/cortx-test-001/pull/1

@aditipande96 @kachhwahadivya

**Pull request closed without merging**